data/json/items/comestibles/other.json

```
@@ -351,7 +351,7 @@
   {
     "type": "COMESTIBLE",
     "id": "can_beans",
-    "name": { "str": "canned beans", "str_pl": "beans" },
+    "name": { "str_pl": "canned beans" },
```

```suggestion
    "name": { "str_sp": "canned beans" },
```
I believe this is the correct format, but it's worth verifying